Update settings support SVG icon

DRadmir opened the pull request:
Replaces the previous SVG illustration for the settings support asset with a new design, updating the background color and icon details for improved appearance and consistency.

<img width="320" alt="Simulator Screenshot - iPhone 17 Pro - 2025-09-19 at 17 28 29" src="https://github.com/user-attachments/assets/be61bf8b-fd43-4a0a-b5ad-d88b4a5f15b0" />

DRadmir committed:
Update settings support SVG icon

Replaces the previous SVG illustration for the settings support asset with a new design, updating the background color and icon details for improved appearance and consistency.

gemcoder21 commented:
Support should be Green I think
<img width="149" height="112" alt="Screenshot 2025-09-20 at 6 22 23 PM" src="https://github.com/user-attachments/assets/55ee12ea-3f9c-4eb4-a571-969b50057dbf" />

DRadmir committed:
Update settings support icon background color

Changed the background color of the settings_support.svg icon from #4746FF to #0AA966 to match updated design requirements.

DRadmir committed:
Merge branch 'main' of github.com:gemwalletcom/gem-ios into update-support-icon

DRadmir commented:
@gemcoder21  updated
<img width="320" alt="Simulator Screenshot - iPhone 17 Pro - 2025-09-22 at 13 32 33" src="https://github.com/user-attachments/assets/cb884a90-1239-46c2-a027-5b64297a11a4" />
